config/settings.yml

```diff
@@ -16,3 +16,7 @@ zendesk:
   defaults:
     group_id: 5455184128796 # 1st Line--GOV.UK Forms
     organization_id: 21891972 # GDS
+
+forms_admin:
+  # URL to form-admin
+  base_url: http://localhost:3000
```

Comment:
I'd like to use this change in my PR for the get started page; can we split it out into it's own PR to merge now?

Reply:
I've done this in https://github.com/alphagov/forms-product-page/pull/113

Reply:
Woops I see this is duplicated at the top of the file...